plugins/infinispan60/src/main/java/org/radargun/service/Infinispan60HotrodService.java

```diff
@@ -71,6 +72,27 @@ public void init() {
    protected ConfigurationBuilder getDefaultHotRodConfig() {
       ConfigurationBuilder builder = new ConfigurationBuilder();
       builder.connectionPool().maxActive(maxConnectionsServer).maxTotal(maxConnectionsTotal);
+      parseServerAddresses().forEach((address) -> builder.addServer().host(address.getHost()).port(address.getPort()));
+      createQueryConfiguration(builder);
+      return builder;
+   }
+
+   protected ConfigurationBuilder createQueryConfiguration(ConfigurationBuilder builder) {
```

Comment:
Have you verified that older plugins are still working with this change?

Reply:
Yes. It was copied and pasted